api/working-with-extensions/publishing-extension.md

```diff
@@ -114,6 +114,13 @@ You can also enter your Personal Access Token as you publish with an optional pa
 vsce publish -p <token>
 ```
 
+Your Personal Access Token may also be defined by the `VSCE_PAT` environmental variable.
+
+```bash
+export VSCE_PAT=97bd38305a81f2d89b5f3aa44500ec964b87cf8a
+vsce publish
+```
+
```

Comment:
I would simply leave this away, since we want users to use the builti-in login functionality instead of using `VSCE_PAT` which is mainly directed to CI.

```suggestion
```

Reply:
Yeah, I take your point. Since there's a cross-reference from this doc to the CI doc, users can come at it from both fronts.

What about something like this:
1. Include a `publish` example to each of the recipes (TravisCI, GitHub Actions) in the existing CI doc
1. Expand the automated publishing section at the bottom of the [publishing page](https://github.com/microsoft/vscode-docs/blob/master/api/working-with-extensions/publishing-extension.md#can-i-publish-from-a-continuous-integration-ci-build) to indicate those other tools have examples, too

I'm not married to the idea, but want to show how this can be easy to do via CI.

Thoughts?

Reply:
I like it, yeah.